webext/data/contentscript.js

```diff
@@ -129,6 +131,12 @@ function clearLocalStorage() {
     }
 }
 
+function olderThanDays(date, days) {
```

Comment:
Doesn't handle DST, but whatevs. Didn't want to import `moment` just for this

Reply:
Reasonable call. 👍